src/app/modules/wallet/components/account-actions/account-actions.component.html

```diff
@@ -21,4 +21,10 @@
       <mat-icon>exit_to_app</mat-icon>
     </button>
   </a>
+  <a class="ml-3">
+    <button class="large-btn ml-1" *ngIf="selection?.selected?.length >0"
+      color="warn"
+      (click)="openDelete()"
+      mat-raised-button>Delete Accounts</button>
```

Comment:
```suggestion
      mat-raised-button>Delete Selected Accounts</button>
```
@rauljordan Do you agree we should add the word `Selected` here? Other buttons have nothing to do with selection, hence the explicitness.